### PR TITLE
RoomTypes list view

### PR DESCRIPTION
**Pull request by kamilsmuga**
- New RoomTypes list view (follows the same UI pattern as Rooms list view)
- jscs suggestions for view/rooms

screen shot 2017-09-20 at 3 17 21 pm

### CODE REVIEW

**Comment by kamilsmuga** [comment]
@tausifmuzaffar found a small bug and fixed it. PR is ready for review

**Comment by tausifmuzaffar** [comment]
Tests still failing

File: view/rooms/list/ListController.js [hunk 1]
```diff
@@ -22,16 +22,25 @@ class ListController extends React.Component {
...
+    rooms.sort((a, b) => moment(b.updatedAt) - moment(a.updatedAt));
+
+    if (rooms.length > 0) {
+      return this.renderRooms(rooms, roomTypes);
+    } else {
+      return this.renderRoomTypes(roomTypes);
+    }
+  }
+
+  renderRooms(rooms, roomTypes) {
```
Comment (tausifmuzaffar) [inline view/rooms/list/ListController.js]: Why create a single component when there are 2 different renders?

I think you need to make this component more generic 

like < ListController headers =[one,two,three] rows=[{1,2,3},{1,23}]>
and from the rooms and room types you parse the data and pass it to the component

Comment (kamilsmuga) [reply view/rooms/list/ListController.js]: agreed. i'll work on it

**Comment by kamilsmuga** [comment]
@tausifmuzaffar requested changes that generify List View implemented. Additionally, I run linter for all view/rooms code and made suggested improvements

File: package.json [hunk 2]
```diff
@@ -106,8 +108,8 @@
     "kue": "^0.11.6",
     "lodash": "^4.0.0",
     "mocha": "^3.1.2",
-    "multer": "^1.0.3",
     "moment": "^2.18.1",
+    "multer": "^1.0.3",
```
Comment (iamigo) [inline package.json]: why?

Comment (kamilsmuga) [reply package.json]: I think npm sorted packages alphabetically automatically

**Comment by kamilsmuga** [comment]
@iamigo screenshot uploaded in the description area of this PR. I followed the same UI pattern as Rooms list view.

File: package.json (hunk 2 above)
Comment (iamigo) [reply package.json]: oh yeah i didn't realize this was *moving* (I just thought you were adding it, and I was like "we are already handling multipart why are you adding this now?" but now I get it)

**Pull request merged**